src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/TestHooks/ValueTaskExtensions.cs

```diff
@@ -0,0 +1,255 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.ExceptionServices;
+using System.Threading.Tasks;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Shared.TestHooks
```

Comment:
Out of curiosity, why is this in a TestHooks namespace?

Reply:
➡️ Matches https://github.com/dotnet/roslyn/blob/main/src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/TestHooks/TaskExtensions.cs

Reply:
Yeah, I saw that. Does it make sense there, or in a better world would it be somewhere else?

Reply:
➡️ Resolving as out-of-scope for this pull request. Please feel free to file an issue.